pkg/plugins/golang/v3/api.go

```diff
@@ -59,6 +59,9 @@ type createAPISubcommand struct {
 
 	// runMake indicates whether to run make or not after scaffolding APIs
 	runMake bool
+
+	// useWorkspaces indicates whether to use go.work style workspaces instead of a go module
+	useWorkspaces bool
```

Comment:
What is go.work?
When and why it is used?
Is that I third party?

Reply:
`go work sync` and `go work init` are inbuilt into the go cli since go 1.18.
You can find the spec here: https://pkg.go.dev/cmd/go/internal/workcmd#:~:text=A%20workspace%20is%20specified%20by%20a%20go.work%20file,builds%20from%20local%20modules.%20go.work%20files%20are%20line-oriented.

Or by using `go help work` which will open the go help for this. I highly recommend it!